### PR TITLE
Create individual event display page

### PR DESCRIPTION
**Pull request by fang-helen**
Created and formatted example display page for an individual event. The right-hand side with the date, time, and save/go back options is fixed, and the left-hand side is scrollable.
Currently, the save event button just links to the "My Events" page, and the go back button will return to index.html.
Additionally, the area for the event type color/background image has a flexible width but a fixed height so it can look a little bit weird on different displays.

### CODE REVIEW

**Review by nmsulliv — APPROVED** [review]

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -260,3 +260,85 @@ h1 {
...
+
+.save-event, .go-back {
+  width: 200px;
+  height: 75px;
+  margin: 15px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+}
```
Comment (0tpyrk) [inline src/main/webapp/style.css]: I believe CSS declarations are supposed to be in alphabetical order according to the style guides

File: src/main/webapp/display-event.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      </div>
+    </div>
+    <div class="event-display-container">
+      <div class="event-left-details">
+        <div class="event-display-image"></div>
+        <div class = "event-left-text">
+          <div class="event-display-header">
+            <div class = "event-display-title">Beach Clean Up</div>
+            <div class="attendee-count-container">
+                    <span class="attendee-count green">12</span> already attending</div>
```
Comment (0tpyrk) [inline src/main/webapp/display-event.html]: The new tag system I created would have this class be "attendee-count environment-text".
The "[tag]-text" class is used to change the color of the text to match what would normally be the tag's background color.

**Review by 0tpyrk — APPROVED** [review]
LGTM, can't wait to see it in action!

File: src/main/webapp/style.css (hunk 1 above)
Comment (fang-helen) [reply src/main/webapp/style.css]: Fixed!

File: src/main/webapp/display-event.html (hunk 2 above)
Comment (fang-helen) [reply src/main/webapp/display-event.html]: Fixed now!

**Pull request merged**